### ISSUE TITLE
Selecting while scrolling doesn't work well in Firefox

### ISSUE THREAD

**Issue by notpushkin** [Simonwep/selection#65]
#### Do you want to request a **feature** or report a **bug**?

Bug

#### What is the current behavior?

When selecting items in a scrolling container in Firefox, only the last screen of items is selected. [Reproduction](https://u.ale.sh/Simonwep-selection-repro.webm)

#### What is the expected behavior?

All items that were previously selected stay selected.

#### Your environment:
```
Selection-version: master probably?
Browser-version: Firefox/71.0
Operating-system: Linux
```

**Comment by Simonwep** [comment]
I think it's fixed in b1577bbbab0bfac3fafc2463d2d5eeaccdf71421, firefox seems to intercept any drag-like mouse-action which messes up the position calculations.

**Comment by notpushkin** [comment]
Works like a charm, thanks!
